docs/examples/fd_fitting_intermediate/index.rst

```diff
@@ -0,0 +1,374 @@
+
+Set up the fit
+--------------
+
+Let's add some custom parameter bounds::
+
+    fit["DNA/Lp"].value = 50
+    fit["DNA/Lp"].lower_bound = 39
+    fit["DNA/Lp"].upper_bound = 80
+
+    fit["DNA/St"].value = 1200
+    fit["DNA/St"].lower_bound = 700
+    fit["DNA/St"].upper_bound = 2000
+
+Fit the model
+-------------
+
+Everything is set up now. All that remains is to do the fit::
+
+    >>> fit.fit()
+
+    Fit
+      - Model: DNA(x-d)_with_DNA
+      - Equation:
+          f(d) = argmin[f](norm(DNA.Lc * (1 - (1/2)*sqrt(kT/(f*DNA.Lp)) + f/DNA.St)-(d - DNA.d_offset))) + DNA.f_offset
+
+      - Data sets:
+        - FitData(Control, N=884)
+        - FitData(RecA, N=1030, Transformations: DNA/Lp → DNA/Lp_RecA, DNA/Lc → DNA/Lc_RecA, DNA/St → DNA/St_RecA)
+
+      - Fitted parameters:
+        Name                 Value  Unit      Fitted      Lower bound    Upper bound
+        ------------  ------------  --------  --------  -------------  -------------
+        DNA/d_offset    -0.0716458  [au]      True               -0.1            0.1
+        DNA/Lp          55.7977     [nm]      True               39             80
+        DNA/Lc           2.83342    [micron]  True                0            inf
+        DNA/St        1407.65       [pN]      True              700           2000
+        kT               4.11       [pN*nm]   False               0              8
+        DNA/f_offset     0.0697629  [pN]      True               -0.1            0.1
+        DNA/Lp_RecA     90.2603     [nm]      True                0            100
+        DNA/Lc_RecA      3.04193    [micron]  True                0            inf
+        DNA/St_RecA    846.33       [pN]      True                0            inf
+
+
+Plot the fit
+------------
+
+Plotting the fit alongside the data is easy. Simply call the plot function on the `FdFit` (i.e. `fit.plot()`)::
+
+    fit.plot()
+    plt.ylabel('Force [pN]')
+    plt.xlabel('Distance [$\\mu$M]')
+
+.. image:: output_10_2.png
+
+We would like to compare the two modelled curves without the data. Plotting these is easy. We can tell the
+model to plot the model for a specific data set by slicing the parameters from our fit with the appropriate
+data handle: `fit[data1]`. This slice procedure collects exactly those parameters needed to simulate that
+condition. The second argument contains the values for the independent variable that we wish to simulate for::
+
+    m_dna.plot(fit[data1], np.arange(2.1, 5.0, .01), 'r--')
+    m_dna.plot(fit[data2], np.arange(2.1, 5.0, .01), 'r--')
+    plt.ylabel('Force [pN]')
+    plt.xlabel('Distance [$\\mu$M]')
+    plt.ylim([0, 30])
+    plt.xlim([2, 3.1])
+
+.. image:: output_10_3.png
+
+Let’s print the contour length difference due to RecA. We multiply by 1000 since we desire this value in
+nanometers::
+
+    >>> print(f"Contour length difference: {(fit['DNA/Lc_RecA'].value - fit['DNA/Lc'].value) * 1000:.2f} [nm]")
```

Comment:
Yes, I will go over all the code parts once more at the very end to make sure it fits.

The only place where I can't guarantee no hori scrollbar is printed output. It might be a nice future improvement to have "nice" line-breaking of equations, but this is a pretty hard problem.

Reply:
Yeah, don't worry about the printed output.

Reply:
👍